hw/top_earlgrey/data/autogen/top_earlgrey.gen.hjson

```diff
@@ -3029,11 +3029,11 @@
           name: RndCnstSramNonce
           desc: Compile-time random reset value for SRAM scrambling nonce.
           type: otp_ctrl_pkg::sram_nonce_t
-          randcount: 64
+          randcount: 96
```

Comment:
This seems to be inconsistent with `SramNonceWidth`, which is only 80 bit wide.
Shall we just go with a 32bit LFSR?

Reply:
ooops let me have a look.